creator-node/compose/env/shellEnv.userMetadata.sh

```diff
@@ -0,0 +1,12 @@
+export COMPOSE_PROJECT_NAME="cn-um"
+export CREATOR_NODE_HOST_PORT=4099
+export CREATOR_NODE_DB_HOST_PORT=4499
+export CREATOR_NODE_REDIS_HOST_PORT=4399
+export CREATOR_NODE_IS_USER_METADATA=true
+
```

Comment:
maybe we should add `snapbackDevModeEnabled=false` here. not super important since it is defaulted to false anyway but could be good to explicitly define that

Reply:
reverted the other change

Reply:
don't we want snapback mode endabled for the content nodes?